anchor-counter-privy/client/.env.local

```
@@ -0,0 +1,4 @@
+REACT_APP_PROVIDER_ENDPOINT=https://api.devnet.solana.com
+REACT_APP_EPHEMERAL_PROVIDER_ENDPOINT=https://devnet.magicblock.app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_EPHEMERAL_PROVIDER_ENDPOINT key should go before the REACT_APP_PROVIDER_ENDPOINT key

(UnorderedKey)